NitroxServer/Player.cs

```diff
@@ -32,6 +32,24 @@ public Player(PlayerContext playerContext, NitroxConnection connection, Vector3
             GameObjectId = new NitroxId();
         }
 
+        /// <summary>
+        /// This overload is used when a NitroxId is already present
+        /// </summary>
+        /// <param name="playerContext"></param>
+        /// <param name="connection"></param>
+        /// <param name="position">the stored position</param>
+        /// <param name="playerId">the stored playerId for the new player</param>
+        /// <param name="subRootId"></param>
```

Comment:
Can we omit empty documentation?

Besides, I don't think `position` and `playerId` are required to come from persisted storage; it is up to the use of this constructor to fill it in with that data. I don't see it necessary to assume that this constructor is only getting called when loading a player from persisted data.